src/Microsoft.Health.Fhir.SqlServer/Features/Search/SqlServerSearchService.cs

```diff
@@ -271,10 +269,18 @@ private async Task<SearchResult> SearchImpl(SearchOptions searchOptions, SqlSear
                             newContinuationId = resourceSurrogateId;
 
                             // Keep track of sort value if this is the last row.
-                            // if we have more than 10 columns, it means sort expressions were added.
+                            // if we have more than 11 columns, it means sort expressions were added.
```

Comment:
>11 [](start = 52, length = 2)

You just updating comment, right?
`_resourceTableColumnCount` is remaining the same, so it's a bit confusing, since that number is based on it.
Maybe it's worth change comment to base it on `_resourceTableColumnCount` instead of number?

Reply:
Updated comment. Also renamed the variable to `_defaultResourceTableFinalSelectColumnCount` from `_resourceTableColumnCount`